src/index.js

```diff
@@ -0,0 +1,146 @@
+    const lat = coordinates.data[0].lat;
+    const lon = coordinates.data[0].lon;
+    const name = coordinates.data[0].display_name;
+    console.log(coordinates);
+    return [lat, lon, name];
+};
+
+const getWeather = async function () {
+    const coordinates = await getCoordinates();
+    const weather = await axios.get(`http://localhost:5000/weather?lat=${coordinates[0]}&lon=${coordinates[1]}`);
+    return weather;
+};
+
+const getSkyData = async function () {
+    const data = await getWeather();
+    const weather = data.data.weather[0].main;
+    return weather;
+};
+
+
+const changeColor = () => {
+    const temp = farenheit.innerText;
+
+    if (temp >= 85) {
+        farenheit.style.color = '#BF360C';
+        emojis.innerHTML = `🌵💀🐍☀️🦂💀🌵☀️🐍🏜☀️🦂`;
+    } else if (temp >= 60) {
+        farenheit.style.color = '#F06292';
+        emojis.innerHTML = `🌸🌿🌼☀️🌷🌻🌿☀️☘️🌱🌻🌷`;
+    } else if (temp >= 45) {
+        farenheit.style.color = 'orange';
+        emojis.innerHTML = `🌾🌾🍃🪨🛤🌾🌾🌾🍃`;
+    } else if (temp <= 40) {
+        farenheit.style.color = "darkBlue";
+        emojis.innerHTML = `🌲❄️⛄️🌲⛄️❄️🌲🍁🌲🌲⛄️❄️`;
+    }
+}
+
+const changeSky = () => {
+
+    if (skyEmoji.value in weatherImages) {
+        sky.innerHTML = weatherImages[skyEmoji.value][0];
+    }
+    else {
+        sky.innerHTML = weatherImages["Other"][0];
+    };
+}
+
+const defaultCity = () => {
+    search.value = "Atlanta";
+    return displayData();
+}
+
+const displayData = async function () {
+    const location = await getCoordinates();
+    const data = await getWeather();
+
+    //display temperature
+    const temperature = data["data"]["main"]["temp"];
+    const tempToFarenheit = Math.round((temperature - 273.15) * 9 / 5 + 32);
+    farenheit.innerHTML = tempToFarenheit;
+
+    //display city and country name
+    //city.innerHTML = `${data["data"]["name"]}, ${data["data"]["sys"]["country"]}`;
+    city.innerHTML = location[2];
+
+    //display sky emojis
+    const skyWeather = await getSkyData();
+
+    if (skyWeather in weatherImages) {
+        sky.innerHTML = weatherImages[skyWeather][0];
+        body.style.background = weatherImages[skyWeather][1];
+    }
+    else {
+        sky.innerHTML = weatherImages["Other"];
+        body.style.background = weatherImages["Other"][1];
+    };
+
+    //display current weather
+    currentWeather.innerHTML = skyWeather;
+
+    //change temp color
+    changeColor();
+};
+
+
+//event listeners 
+searchBtn.addEventListener("click", function (evt) {
+    evt.preventDefault();
+    city.innerText = search.value;
+    displayData();
+    changeColor();
+
+})
+
+skyEmoji.addEventListener("change", changeSky);
+
+increaseBtn.addEventListener("click", function (evt) {
+    farenheit.innerHTML++;
+    changeColor();
+});
+
+decreaseBtn.addEventListener("click", function (evt) {
+    farenheit.innerHTML--;
+    changeColor();
+})
+
+reset.addEventListener("click", defaultCity);
+
+body.onload = defaultCity;
+
```

Comment:
Only suggestion here would be to move your `addEventListeners` for your smaller elements into a `document.addEventListener('DOMContentLoaded', () => {})`. Finally, great work as usually Tazmeen, it's always a pleasure to go through your code! ✨✨✨